### DISCUSSION TITLE
Application for student pack stucked in a cycle.

### DISCUSSION THREAD

**Discussion by AyushBhosale** [community/community#138183] [GitHub Education]
I am a student who's college does not provide college-gmail so I added my contact gmail ID to my GitHub account and verified it but the application process is locked in a cycle.

I tried to submit my application on https://education.github.com/discount_requests/application?type=student. After that as per instruction I added my gmail to my account. After which I revisited the application page which was asking to write college name which I did but the same request of add email Id is popping even after I added my contact email which has been asked please take a note of this need GitHub's student pack benefits. I have been frustrated for past 2 hours please take a note and suggest me solution for this problem.
Yours Sincerely,
Ayush Bhosale

**Comment by apurva313** [comment]
> I am a student who's college does not provide college-gmail so I added my contact gmail ID to my GitHub account and verified it but the application process is locked in a cycle.
> 
> I tried to submit my application on https://education.github.com/discount_requests/application?type=student. After that as per instruction I added my gmail to my account. After which I revisited the application page which was asking to write college name which I did but the same request of add email Id is popping even after I added my contact email which has been asked please take a note of this need GitHub's student pack benefits. I have been frustrated for past 2 hours please take a note and suggest me solution for this problem. Yours Sincerely, Ayush Bhosale


If your college did not provide a college email ID, you can use your college ID card and upload a picture of it taken with a camera. Your application will be approved within 2 days.[](url)

**Reply by AyushBhosale** [reply]
welp for that I need to go through this
![Screenshot 2024-09-11 001450](https://github.com/user-attachments/assets/2e518ba0-39c9-4081-ab77-6ab9a82af0e5)
but I have already added emails
![Screenshot 2024-09-11 001504](https://github.com/user-attachments/assets/29d7537a-8262-4c90-b991-1addaeb6af03)
but now when I refresh the page nothing goes through

**Reply by AyushBhosale** [reply]
Github community please help me
